cmake/Modules/FindMPLIB.cmake

```diff
@@ -11,7 +11,7 @@ if(GMP_FOUND)
     set(MPLIB_PC_ADD_LIBS ${GMP_PC_ADD_LIBS})
 else(GMP_FOUND)
     message(STATUS "GMP not found; this is not a problem if MPIR can be found.")
-    find_package(MPIR REQUIRED)
+    find_package(MPIR)
```

Comment:
Will this fall through if not found, giving the "MPIR found" message and doing all the sets below?

Reply:
Hm, I think I'll clean up this section.